test/collections.js

```diff
@@ -826,6 +826,8 @@
     assert.equal(_.size({one: 1, two: 2, three: 3}), 3, 'can compute the size of an object');
     assert.equal(_.size([1, 2, 3]), 3, 'can compute the size of an array');
     assert.equal(_.size({length: 3, 0: 0, 1: 0, 2: 0}), 3, 'can compute the size of Array-likes');
+    assert.equal(_.size(new Set([1, 22, 55, 12])), 4, 'can compute the size of an Set');
```

Comment:
IE11 doesn't support map/set initialization through arrays in the constructor so you'll have to create them like
```js
var set = new Set;
set.add(1)
set.add(22);

var map = new Map;
map.set(1, 5);
map.set(3, 4);
```